templates/lxd/index.html

```diff
@@ -0,0 +1,299 @@
+      <ul class="p-list--divided u-no-padding--bottom">
+        <li class="p-list__item is-ticked">Flexible resource limits (CPU, memory, network I/O, disk space, and some kernel resources)</li>
+        <li class="p-list__item is-ticked">Advanced snapshot support, including scheduling and automatic expiry</li>
+        <li class="p-list__item is-ticked">Projects for segmenting your LXD server, and easy multi-user setup for enhanced security</li>
+        <li class="p-list__item is-ticked">Hardware passthrough (GPU, USB, NIC, disks, and more)</li>
+        <li class="p-list__item is-ticked">Support for live-migration and stateful snapshotting</li>
+        <li class="p-list__item is-ticked">Advanced networking support (OVN, SR-IOV, hardware acceleration  support)</li>
+        <li class="p-list__item is-ticked">High availability clustering when combined with CEPH and OVN for storage and network redundancy</li>
+        <li class="p-list__item is-ticked">LTS releases every two years &mdash; supported for five years</li>
+        <li class="p-list__item is-ticked">Commercial support available through <a href="https://ubuntu.com/pro">Ubuntu Pro</a></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section--shallow">
+  <hr class="p-rule is-fixed-width">
+  <div class="u-fixed-width">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/ca68d7e3-stocksy-woman-working-ar2-1.jpg",
+      alt="",
+      width="2048",
+      height="836",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
+    }}
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--50-50">
+    <div class="col">
+      <h3 class="p-heading--2">Join the LXD community</h3>
+    </div>
+    <div class="col">
+      <div class="row col-medium-2">
+        <div class="col-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Documentation</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://documentation.ubuntu.com/lxd/en/latest/">Read the documentation&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule is-muted">
+      <div class="row col-medium-2">
+        <div class="col-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Forum</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://discourse.ubuntu.com/c/lxd/126">Discuss with other users&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule is-muted">
+      <div class="row col-medium-2">
+        <div class="col-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Tutorials</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://www.youtube.com/c/LXDvideos/videos">Watch tutorials on YouTube&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
```

Comment:
The order of these 2 sections is swapped in the design: 
![image](https://github.com/canonical/canonical.com/assets/17607612/3e00bef7-180c-4eda-aee0-b8e45f8eb0b1)

Reply:
This was design +1d, so I think @lyubomir-popov is ok with this order or maybe he changed it?

Reply:
put the image to the section where it's supposed to be by the design